installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -14,20 +16,17 @@ vagrant_dev_box_add() {
     # Returns: $box
     box=$1
     declare provider=virtualbox
-    if [[ $_vagrant_dev_host_os == ubuntu ]]; then
-        provider=libvirt
-    fi
     if [[ $vagrant_dev_box ]]; then
         box=$vagrant_dev_box
     elif [[ $box =~ fedora ]]; then
         if [[ $box == fedora ]]; then
             box=generic/fedora$install_version_fedora
         fi
-    elif [[ $box == centos ]]; then
+    elif [[ $box == centos || $box == alma ]]; then
```

Comment:
Make backwards compatible. Convert the argument on entry. Centralize the name mapping in a separate function.

In main we except alma* which could mean almalinux. I don't know what the right name is. This code should be abstracted to return the box name from the arg, which is an abstract entity. The parsing shouldn't be here.